server/plugin/command.go

```diff
@@ -625,3 +631,12 @@ func parseCommand(input string) (command, action string, parameters []string) {
 
 	return command, action, parameters
 }
+
+func Contains(a []string, x string) bool {
```

Comment:
Maybe to differentiate from `strings.Contains`

```suggestion
func SliceContainsString(a []string, x string) bool {
```

Reply:
Thanks for the suggestion! Changed it now